Rakefile

```diff
@@ -1,24 +1,26 @@
 #!/usr/bin/env rake
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
 
 begin
+  require 'rubocop/rake_task'
+  RuboCop::RakeTask.new
+
   require 'rspec/core/rake_task'
 
-  desc "Run Integration Tests"
+  desc 'Run Integration Tests'
   RSpec::Core::RakeTask.new(:integration) do |t|
     t.pattern = FileList["spec/integration/**/*#{ENV['PATTERN']}*_spec.rb"]
   end
 
-
-  desc "Run Unit Tests"
+  desc 'Run Unit Tests'
   RSpec::Core::RakeTask.new(:spec) do |t|
     t.pattern = FileList["spec/lib/auth0/**/*#{ENV['PATTERN']}*_spec.rb"]
   end
 
-  desc "Run All Suites"
+  desc 'Run All Suites'
   RSpec::Core::RakeTask.new(:all)
 
-  task :default => :spec
+  task default: [:rubocop, :spec]
 rescue LoadError
-  #No RSpec
+  puts 'Nothing to do'
```

Comment:
Rubocop does not allow to have a rescue with no code inside. I put the "puts " but I think this should be fixed. 
Any suggestions?